src/api/spec/factories/packages.rb

```diff
@@ -1,10 +1,12 @@
 FactoryGirl.define do
   factory :package do
+    project
     sequence(:name) { |n| "package_#{n}" }
     factory :package_with_file do
       after(:create) do |package|
         # NOTE: Enable global write through when writing new VCR cassetes.
         # ensure the backend knows the project
+CONFIG['global_write_through'] = true
```

Comment:
Is this a leftover? If you needed it at developing time in the controller spec... then let it there commented like in other specs that we have done